libraries/rush-lib/webpack.config.js

```diff
@@ -22,31 +23,26 @@ module.exports = () => {
     ...Object.keys(packageJson.devDependencies || {})
   ]);
 
-  return {
+  const configuration = {
+    context: __dirname,
     mode: 'development', // So the output isn't minified
     devtool: 'source-map',
     entry: {
```

Comment:
Where's the actual entry point to the bundle? If it's coming from somewhere else, that needs a comment.